src/test/resources/alma-fix/(DE-605)HT019868557.json

```diff
@@ -50,12 +50,9 @@
     "id" : "http://hub.culturegraph.org/resource/HBZ-HT019868557",
     "label" : "Culturegraph Ressource"
   } ],
-  "isPartOf" : [ {
-    "type" : [ "IsPartOfRelation" ],
-    "hasSuperordinate" : [ {
-      "id" : "http://lobid.org/resources/HT019807972#!",
-      "label" : "<<Der>> Rhein / herausgegeben von/édité par Hélène Miard-Delacroix, Guido Thiemeyer"
-    } ]
+  "containedIn" : [ {
+    "id" : "http://lobid.org/resources/HT019807972#!",
+    "label" : "lobid Ressource"
```

Comment:
Wouldn't it be nice to have the label, maybe even with the whole information concatenated? The source xml shows:
```
 <subfield code="i">Enthalten in</subfield>
 <subfield code="t">&lt;&lt;Der&gt;&gt; Rhein / herausgegeben von/&#233;dit&#233; par H&#233;l&#232;ne Miard-Delacroix, Guido Thiemeyer</subfield>
 <subfield code="d">Stuttgart, 2018</subfield>
 <subfield code="g">Seite 219-231</subfield>
 <subfield code="k">Schriftenreihe des deutsch-franz&#246;sischen Historikerkomitees ; Band 14</subfield>
```
We _could_  leave this out - anyone (even our one web frontend) could load the data on the fly to present it - but in general the `label` is considered to help frontends to show something meaningful in a simple way.

Reply:
Could do this in an additional PR. $t is the name of the containing ressource

Reply:
Ok, opened issue https://github.com/hbz/lobid-resources/issues/1410.